CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 This CHANGELOG follows the format listed [here](https://github.com/sensu-plugins/community/blob/master/HOW_WE_CHANGELOG.md)
 
+## Unreleased
```

Comment:
`[Unreleased]` so it can be linked to. You should probably also create the missing link(s) at the bottom of the changelog.